internal/client/model.go

```diff
@@ -2341,3 +2341,77 @@ type DeleteSecretRotationRequest struct {
 type DeleteSecretRotationResponse struct {
 	SecretRotation SecretRotation `json:"secretRotation"`
 }
+
+type ProjectTemplate struct {
+	ID           string        `json:"id"`
+	Name         string        `json:"name"`
+	Description  string        `json:"description"`
+	Roles        []Role        `json:"roles"`
+	Environments []Environment `json:"environments"`
+	OrgID        string        `json:"orgId"`
+	CreatedAt    time.Time     `json:"createdAt"`
+	UpdatedAt    time.Time     `json:"updatedAt"`
+	Type         string        `json:"type"`
+}
+
+type Environment struct {
+	Name     string `json:"name" tfsdk:""`
+	Slug     string `json:"slug"`
+	Position int64  `json:"position"`
+}
+
+type PermissionCondition struct {
+	Environment string `json:"environment"`
+	SecretPath  string `json:"secretPath"`
+	SecretName  string `json:"secretName"`
+}
+
+type Role struct {
+	Name        string       `json:"name"`
+	Slug        string       `json:"slug"`
+	Permissions []Permission `json:"permissions"`
+}
+
+type Permission struct {
+	Subject   string               `json:"subject"`
+	Action    []string             `json:"action"`
+	Condition *PermissionCondition `json:"conditions,omitempty"`
```

Comment:
Condition here should be a generic `map[string]any` right?

<img width="1704" height="1190" alt="image" src="https://github.com/user-attachments/assets/bc1c3a7c-6d82-49a4-b491-54c96ed5e364" />